charts/jaeger-v2/values.yaml

```diff
@@ -0,0 +1,71 @@
+
+  service:
+    extensions: [jaeger_storage, jaeger_query, remote_sampling, healthcheckv2]
+    pipelines:
+      traces:
+        receivers: [otlp, jaeger, zipkin]
+        processors: [batch, adaptive_sampling]
+        exporters: [jaeger_storage_exporter]
+
+  extensions:
+    healthcheckv2:
+      use_v2: true
+      http: {}
+
+    jaeger_query:
+      storage:
+        traces: some_store
+        traces_archive: another_store
+      ui:
+        config_file: ./cmd/jaeger/config-ui.json
+
+    jaeger_storage:
+      backends:
+        some_store:
+          memory:
+            max_traces: 100000
+        another_store:
+          memory:
+            max_traces: 100000
+
+    remote_sampling:
+      adaptive:
+        sampling_store: some_store
+        initial_sampling_probability: 0.1
```

Comment:
I don't think we want adaptive sampling to be the default.

Reply:
i tried removing it but it gives me this error 
```2024/10/11 11:48:44 application version: git-commit=4b7446248e087edffd15508e760e8e5da044f4b4, git-version=v2.0.0-rc2, build-date=2024-10-07T09:16:33Z
Error: invalid configuration: extensions::remote_sampling: no sampling strategy provider specified, expecting 'adaptive' or 'file'
2024/10/11 11:48:44 invalid configuration: extensions::remote_sampling: no sampling strategy provider specified, expecting 'adaptive' or 'file'```

Reply:
remove remote sampling extension altogether to start with, let's get a real storage working first.